ui/src/app/pages/access/modules-for-annual-renewal.tsx

```diff
@@ -1,18 +1,482 @@
 import * as React from 'react';
```

Comment:
is this all copy-paste from annual-renewal?

Reply:
Yeah, this is copy and paste.